pkg/ddevapp/wordpress.go

```diff
@@ -184,7 +286,7 @@ func setWordpressSiteSettingsPaths(app *DdevApp) {
 	settingsFileBasePath := filepath.Join(app.AppRoot, app.Docroot)
 	var settingsFilePath, localSettingsFilePath string
 	settingsFilePath = filepath.Join(settingsFileBasePath, "wp-config.php")
-	localSettingsFilePath = filepath.Join(settingsFileBasePath, "wp-config-local.php")
+	localSettingsFilePath = filepath.Join(settingsFileBasePath, "wp-config-ddev.php")
```

Comment:
I can rename the var `localSettingsFilePath` to `ddevSettingsFilePath`, that makes sense.

It will not break anything necessarily, because additional/environment configs requires manual work in `wp-config.php` to be included in the first place. Before these changes when DDEV created `wp-config-local.php` it was not be used unless a user added it by hand.

So `wp-config-local.php` is a non-standard WP way to handle multiple environments. If it exist, and we didn't create it. It's being used for a different local setup most likely. The user has to do work to include and additional configs by hand in the `wp-config.php`. That's why I felt for users trying out DDEV with an existing site, this naming convention made more sense. I believe passed on past PR's I have sen from @cweagans I feel other team members felt the same way.

Environment Based  WP Config Examples:
https://abandon.ie/notebook/wordpress-configuration-for-multiple-environments
https://jonsuh.com/blog/configure-wordpress-for-multiple-environments/
https://github.com/studio24/wordpress-multi-env-config
https://github.com/Abban/WordPress-Config-Bootstrap/blob/master/wp-config.php
https://gist.github.com/ahaywood/c11f1d756521c3edee9a822c0b9cccc7

I'm totally open to including both or using `wp-config-local.php`. Ideally we use env vars.